spec/forms/work_form_spec.rb

```diff
@@ -6,15 +6,28 @@
 RSpec.describe WorkForm do
   subject(:form) { described_class.new(work) }
 
-  let(:work) { build(:work) }
+  let(:work) { create(:work) }
```

Comment:
I'm pretty sure this is unnecessary.

Reply:
Yep. That was from a prior bit of flailing. Reverting that change.